packages/common/src/dbt/validation.ts

```diff
@@ -11,31 +13,34 @@ import { ParseError } from '../types/errors';
 import lightdashMetadataSchema from './schemas/lightdashMetadata.json';
 import lightdashSchemaV10 from './schemas/lightdashV10.json';
 import lightdashSchemaV11 from './schemas/lightdashV11.json';
+import lightdashSchemaV12 from './schemas/lightdashV12.json';
 import lightdashSchemaV7 from './schemas/lightdashV7.json';
 import lightdashSchemaV8 from './schemas/lightdashV8.json';
 import lightdashSchemaV9 from './schemas/lightdashV9.json';
 import dbtManifestSchemaV10 from './schemas/manifestV10.json';
 import dbtManifestSchemaV11 from './schemas/manifestV11.json';
+import dbtManifestSchemaV12 from './schemas/manifestV12.json';
 import dbtManifestSchemaV7 from './schemas/manifestV7.json';
 import dbtManifestSchemaV8 from './schemas/manifestV8.json';
 import dbtManifestSchemaV9 from './schemas/manifestV9.json';
 
-const ajv = new Ajv({
-    schemas: [
-        dbtManifestSchemaV7,
-        dbtManifestSchemaV8,
-        dbtManifestSchemaV9,
-        dbtManifestSchemaV10,
-        dbtManifestSchemaV11,
-        lightdashMetadataSchema,
-        lightdashSchemaV7,
-        lightdashSchemaV8,
-        lightdashSchemaV9,
-        lightdashSchemaV10,
-        lightdashSchemaV11,
-    ],
-});
-
+const ajv = new Ajv2020();
+ajv.addMetaSchema(draft7MetaSchema); // add backward compatibility with draft-07
+ajv.addSchema([
```

Comment:
**draft 2020** is not compatible with **draft 07**
So we need to initialize AVJ2020 and add draft 07 meta schema.